source/03_system/config.rst

```diff
@@ -0,0 +1,127 @@
+
+  - Fields that exist in the default configuration but not in the user configuration will use the default values.
+  - Fields that exist in the user configuration but not in the default configuration will use the values specified by the user.
+  - Fields that exist in both the default configuration and the user configuration will use the values specified by the user.
+
+The Categories of Core Configuration Fields of a Reinforcement Learning Task
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Overall, the configuration fields required for reinforcement learning tasks in DI-engine can be mainly classified into two categories:
+
+- The first category consists of objects that are shared by various workers/middleware in the training pipeline, such as algorithm policies (Policy), environments (Env), reward models (Reward Model), data queues (Buffer), and so on. Each of these objects has its own default configuration. In the overall configuration file, they are placed in parallel with each other and maintain a flat hierarchy. The meaning of each object's configuration fields can be found in the corresponding class definition, and the class variables ``config`` and class method ``default_config`` specify the default configuration and calling method.
+
+- The second category consists of workers/middleware that perform various tasks in the training pipeline, such as learners (Learner), data collectors (Collector), and so on. They generally have fewer corresponding parameters, which can be directly specified in the training entry function or attached to the global area of the overall configuration file. There are no special requirements for their specific implementation, and the code should be kept clear and easy to use.
+
+In addition to the main configuration ``main_config``, there is also a creation configuration ``create_config`` that helps to quickly create training entry points. This part of the creation configuration is only used in the ``serial_pipeline_xxx`` series of quick training entry points, and can be ignored by users who create their own custom training entry functions. The creation configuration needs to specify the corresponding type name (type) and the path required to import the module (import_names).
+
+.. note::
+  Due to historical reasons, there were many sub-domains defined in the Policy field, such as learn, collect, eval, other, and so on. However, in the latest version of DI-engine (>=0.4.7), the mandatory dependency on these definitions has been removed. Now, using or not using these structures is optional, as long as the configuration fields are corresponded with the corresponding effective code segments in the policy.
+
+
+Other Tools
+^^^^^^^^^^
+DI-engine also provides some related tools for configuration file storage and formatting. For specific information, please refer to the code. `ding/config <https://github.com/opendilab/DI-engine/tree/main/ding/config>`_
+
```

Comment:
please refer to the code `ding/config <https://github.com/opendilab/DI-engine/tree/main/ding/config>`_.